app/controllers/v0/disability_compensation_forms_controller.rb

```diff
@@ -6,13 +6,24 @@
 require 'evss/disability_compensation_form/service'
 require 'evss/reference_data/service'
 require 'evss/reference_data/response_strategy'
+require 'disability_compensation/factories/api_provider_factory'
 
 module V0
   class DisabilityCompensationFormsController < ApplicationController
     before_action { authorize :evss, :access? }
     before_action :validate_name_part, only: [:suggested_conditions]
 
     def rated_disabilities
+      service_sym = if Flipper.enabled?('disability_compensation_lighthouse_rated_disabilities_provider')
```

Comment:
I feel like the flipper check should be done inside the factory, that way if want to use this in other places, we don't have to duplicate this code in those other places. I can't remember if rated disabilities is used in other places, but other migrations we will do are and this could set the precedent of the factory making that decision and not the user of the factory.

Reply:
Now that you mention it, putting Flipper logic into a module/service feels better to me than putting it in a controller - even when the relevant API is only used in one place.. So I would second the motion.